bazel/linux/qemu.bzl

```diff
@@ -28,6 +33,10 @@ def _kernel_qemu_run(ctx):
 QEMU_FLAGS={qemu_flags}
 QEMU_SEARCH={qemu_search}
 KERNEL_FLAGS={kernel_flags}
+BOOTRAM_IMAGE={bootram_image}
+ARCH={arch}
+
+echo $ARCH
```

Comment:
Is this `echo` for debug?  If so, remove it.

Reply:
Yeah removed it.